.github/workflows/compare-js-runtimes.yml

```diff
@@ -0,0 +1,153 @@
+name: Run and compare benchmarks on different JS runtimes
```

Comment:
Created this workflow as reusable workflow so we can utilize it in other repos.

Reply:
This is pretty sweet.  Do we need some docs on how to use it?  Maybe a comment block at the head for consumers that goes through what it does and how to run it so they dont have to study the code to grok whats cooking